jobs/base_model_validator_errors_test.py

```diff
@@ -34,6 +34,10 @@ class MockModel(base_models.BaseModel):
     pass
 
 
+class MockCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
```

Comment:
Please use the real model instead, and delete the mock.

Reply:
Done